docs/howto/deployment/vercel-netlify.md

```diff
@@ -62,12 +62,18 @@ yum install wget -y
 
 wget -qO- https://micromamba.snakepit.net/api/micromamba/linux-64/latest | tar -xvj bin/micromamba
```

Comment:
Maybe we can also update the link to the binary with the latest from the docs: https://mamba.readthedocs.io/en/latest/installation/micromamba-installation.html

```suggestion
wget -qO- https://micro.mamba.pm/api/micromamba/linux-64/latest | tar -xvj bin/micromamba
```